bosh-setup/azuredeploy.json

```diff
@@ -66,6 +66,7 @@
   },
   "variables": {
     "apiVersion": "2015-06-15",
```

Comment:
I think it's a good way to define a variable `apiVersion`, and use it everywhere.
What does the intellisense of json editors mean? Which json editor?
Which one is the best practice? Defining a variable `apiVersion` or hard-coding the apiVersion in the resources?